Bottom: 416731e3a74dbbc6525bc8fd8d96b1de9fed7498
Top:    2d2951258f22895a7bac3e1b5204eb0836ad9405
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-09-10 17:11:56 +0100

Refresh of major-re-write-to-drawing-rout

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 273649a..1e1393b 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1529,12 +1529,12 @@ ghid_draw_everything (BoxTypePtr drawn_area)
     reverse_layers = !reverse_layers;
 
   memset (do_group, 0, sizeof (do_group));
-  for (ngroups = 0, i = 0; i < max_group; i++) {
+  for (ngroups = 0, i = 0; i < max_copper_layer; i++) {
     LayerType *l;
     int group;
     int orderi;
 
-    orderi = reverse_layers ? max_group - i - 1 : i;
+    orderi = reverse_layers ? max_copper_layer - i - 1 : i;
 
     // Draw in numerical order when in 3D view
     l = global_view_2d ? LAYER_ON_STACK (i) : LAYER_PTR (orderi);
